application/cloud_api.py

```diff
@@ -3,6 +3,7 @@
 import pickle
 import uuid
 import requests
+from requests_toolbelt import MultipartEncoder
```

Comment:
you should rebase dev (now that Qt5 branch is merged) and add requests_toolbelt to the install scripts